plugins/lb302/lb302.cpp

```diff
@@ -433,8 +433,11 @@ QString lb302Synth::nodeName() const
 // OBSOLETE. Break apart once we get Q_OBJECT to work. >:[
 void lb302Synth::recalcFilter()
 {
-	vcf.load()->recalc();
-
+#if (QT_VERSION >= QT_VERSION_CHECK(5,14,0))
+	vcf.loadRelaxed()->recalc();
+#else
+	vcf.load()->playNote();
```

Comment:
Shouldn't this line be `vcf.load()->recalc();`?

Reply:
Oh yes, my bad.